cli/templateedit_test.go

```diff
@@ -103,4 +93,33 @@ func TestTemplateEdit(t *testing.T) {
 		assert.Equal(t, template.MaxTTLMillis, updated.MaxTTLMillis)
 		assert.Equal(t, template.MinAutostartIntervalMillis, updated.MinAutostartIntervalMillis)
 	})
+	t.Run("InvalidDisplayName", func(t *testing.T) {
+		t.Parallel()
+		client := coderdtest.New(t, &coderdtest.Options{IncludeProvisionerDaemon: true})
+		user := coderdtest.CreateFirstUser(t, client)
+		version := coderdtest.CreateTemplateVersion(t, client, user.OrganizationID, nil)
+		_ = coderdtest.AwaitTemplateVersionJob(t, client, version.ID)
+		template := coderdtest.CreateTemplate(t, client, user.OrganizationID, version.ID)
+
+		// Test the cli command.
+		cmdArgs := []string{
+			"templates",
+			"edit",
+			template.Name,
+			"--name", template.Name,
+			"--display-name", "a-b-c",
+		}
+		cmd, root := clitest.New(t, cmdArgs...)
+		clitest.SetupConfig(t, client, root)
+
+		err := cmd.Execute()
```

Comment:
Considering using e.g. `testutil.Context` and passing it to `cmd.ExecuteContext`. This is to ensure we don't have singular test hanging only the fail CI/test suite after a long timeout.

Reply:
Fixed